### DISCUSSION TITLE
横向的菜单,hover显示隐藏的闪动的BUG，在官方文档和自己项目中都存在该问题

### DISCUSSION THREAD

**Discussion by wangrongding** [element-plus/element-plus#8188] [Q&A]
当鼠标点击子菜单后，菜单开始闪动。

![czxczxc](https://user-images.githubusercontent.com/42437658/172623514-b40889f6-f5b5-4439-aee3-4e8efe343520.gif)

另外提不了issues了，因为选择框没有选项，选不了但是又是必填项。。。😅

![image](https://user-images.githubusercontent.com/42437658/172624114-f1cba982-7ad7-4056-b304-d0a790fc59f6.png)

**Comment by btea** [comment]
fixed #8533
